### PR TITLE
Don't update search domains

### PR DESCRIPTION
**Pull request by viovanov**

### CODE REVIEW

**Comment by mook-as** [comment]
Some context:

On GKE, we're having issues due to:
> Warning  FailedCreate  9m3s   job-controller  Error creating: Pod "app-saalok-2hx7w" is invalid: spec.dnsConfig.searches: Invalid value: []string{"scf.svc.cluster.local", "svc.cluster.local", "cluster.local", "europe-west3-c.c.blah.internal", "c.blah.internal", "google.internal", "service.cf.internal"}: must not have more than 6 search paths 

Essentially, then, this extension is down to clobbering the DNS nameservers to use the custom one? Can we rolle it into [kubecf-apps-dns](https://github.com/SUSE/kubecf-apps-dns) instead then? This isn't really bringing any value anymore.

File: pkg/extension/extension.go [hunk 1]
```diff
@@ -69,7 +60,6 @@ func (ext *Extension) Handle(
 	podCopy.Spec.DNSConfig = &corev1.PodDNSConfig{
 		Nameservers: nameservers,
 		Options:     options,
-		Searches:    searches,
```
Comment (mook-as) [inline pkg/extension/extension.go]: Instead of not updating, I believe this just plain clobbers any search domains instead… which can't be great?

**Review by mook-as — CHANGES_REQUESTED** [review]

File: pkg/extension/extension.go (hunk 1 above)
Comment (viovanov) [reply pkg/extension/extension.go]: good catch, thx

**Review by mook-as — APPROVED** [review]
Ideally we'd test the searches modification too…

**Pull request merged**